warp/cairo-src/evm/uint256.cairo

```diff
@@ -87,6 +87,10 @@ func is_eq{range_check_ptr}(op1 : Uint256, op2 : Uint256) -> (result : Uint256):
 end
 
 func is_gt{range_check_ptr}(op1 : Uint256, op2 : Uint256) -> (result : Uint256):
+    let (eq) = uint256_eq(op1, op2)
+    if eq == 1:
+        return (result=Uint256(0, 0))
+    end
```

Comment:
I think in this case you do not need to check for equality. You want to know if `op1 > op2` and you are returning `op2 < op1` when you do `let (res) = uint256_lt(op2, op1)`, which is supposed to be the same thing. Am I correct?

Reply:
Yeah, I think this whole function should be `return is_lt(op2, op1)`